src/pom.xml

```diff
@@ -1475,6 +1475,67 @@
      </configuration>
    </plugin>
 
+<!-- ======================================================= -->
+   <!--     Javadoc generation.                                 -->
+   <!-- ======================================================= -->
+   <plugin>
+    <artifactId>maven-javadoc-plugin</artifactId>
+    <version>2.10.3</version>
+    <configuration>
+     <source>1.8</source>
+     <version>false</version>
+     <noqualifier>all</noqualifier>
+     <quiet>true</quiet>
+     <maxmemory>256M</maxmemory>
+     <encoding>UTF-8</encoding>
+     <!-- <additionalparam>-keywords</additionalparam> -->
+     <additionalparam>-Xdoclint:none</additionalparam>
+     <breakiterator>true</breakiterator>
+     <tags>
+      <tag>
+       <name>todo</name>
+       <placement>tfmc</placement>
+       <head>TODO:</head>
+      </tag>
+      <tag>
+       <name>task</name>
+       <placement>tfmc</placement>
+       <head>TODO:</head>
+      </tag>
+      <tag>
+       <name>revisit</name>
+       <placement>tfmc</placement>
+       <head>TODO:</head>
+      </tag>
+      <tag>
+       <name>source</name>
+       <placement>Xt</placement>
+       <head>Source:</head>
+      </tag>
+      <tag>
+       <name>tutorial</name>
+       <placement>Xt</placement>
+       <head>Tutorial:</head>
+      </tag>
+     </tags>
+
+     <links>
+      <link>http://docs.oracle.com/javase/7/docs/api/</link>
```

Comment:
my other pull request updates some of these URLS to point at java 8 (and removes dead URLs for JAI)